server/db/connection.js

```diff
@@ -2,9 +2,11 @@ require("dotenv").config();
 const mongoose = require("mongoose");
 const db = require("./models");
 
+const { password } = require("./password");
```

Comment:
This will also need to be updated to process.env when you move the password to the .env